src/client/app/actions/meters_filter.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+export const METERS_FILTER_MODIFIED = 'METERS_FILTER_MODIFIED';
```

Comment:
Why do we have Redux actions for filter out meters/groups? This could all be done in a single React component.